packages/design-system/src/contexts/snackbar/snackbarProvider.tsx

```diff
@@ -42,15 +42,15 @@ function SnackbarProvider({
   const remove = useCallback(
     (
       toRemove:
-        | Pick<SnackbarNotification, 'key'>
-        | Pick<SnackbarNotification, 'key'>[]
+        | Pick<SnackbarNotification, 'id'>
+        | Pick<SnackbarNotification, 'id'>[]
     ) => {
       setNotifications((currentNotifications) =>
         currentNotifications.filter((item) => {
           if (Array.isArray(toRemove)) {
-            return !toRemove.find(({ key }) => key === item.key);
+            return !toRemove.find(({ id }) => id === item.id);
           }
-          return item.key !== toRemove.key;
+          return item.id !== toRemove.id;
```

Comment:
`key` is actually used in at least one place:

https://github.com/GoogleForCreators/web-stories-wp/blob/be47067e36ec9d7c2b2d5092006953d3937fa9f1/packages/story-editor/src/app/media/useUploadMedia.js#L106-L116

Should we just remove `key` there because we only show 1 notification now anyway?

If so, we should probably remove `key` entirely from the `SnackbarNotification` type.

Reply:
instead of using `key` here we can pass `id` in `showSnackbar`. This will overwrite the default `id` given using `uuid`.
In case we want to show more than one snack bar in the future, notifications can be still deleted using `removeSnakbar` and a known id.